### PR TITLE
New features: Turn maintenance mode on or off

### PR DESCRIPTION
**Pull request by chiyi4488**
This feature is designed for users who are just using nextcloud and can simply troubleshoot problems (such as accidentally opening maintenance mode when updating).

Thanks

### CODE REVIEW

File: bin/ncp/TOOLS/nc-maintenance.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# GPL licensed (see end of file) * Use at your own risk!
+#
+# More at: https://www.cotpear.com
+# Made in Taiwan (Republic of China)
+#
+
+configure()
+{
+  [[ $ACTIVE != "yes" ]] && {
+    sudo -u www-data php /var/www/nextcloud/occ maintenance:mode --off
```
Comment (nachoparker) [inline bin/ncp/TOOLS/nc-maintenance.sh]:
```suggestion
    ncc maintenance:mode --off
```

File: bin/ncp/TOOLS/nc-maintenance.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    return 0 
+  }
+  sudo -u www-data php /var/www/nextcloud/occ maintenance:mode --on
```
Comment (nachoparker) [inline bin/ncp/TOOLS/nc-maintenance.sh]:
```suggestion
  ncc maintenance:mode --on 
```

File: etc/ncp-config.d/nc-maintenance.cfg [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "id": "nc-maintenance",
+  "name": "Nc-maintenanceMode",
+  "title": "nc-maintenance-mode",
+  "description": "Turn maintenance mode on or off",
+  "info": "Open maintenance mode (ACTIVE=yes), or close maintenance mode  (ACTIVE=no)",
```
Comment (nachoparker) [inline etc/ncp-config.d/nc-maintenance.cfg]:
```suggestion
  "info": "Open maintenance mode (ACTIVE=yes), or close maintenance mode (ACTIVE=no)",
```

File: bin/ncp/TOOLS/nc-maintenance.sh (hunk 1 above)
Comment (chiyi4488) [reply bin/ncp/TOOLS/nc-maintenance.sh]: changed

File: bin/ncp/TOOLS/nc-maintenance.sh (hunk 2 above)
Comment (chiyi4488) [reply bin/ncp/TOOLS/nc-maintenance.sh]: changed

File: etc/ncp-config.d/nc-maintenance.cfg (hunk 3 above)
Comment (chiyi4488) [reply etc/ncp-config.d/nc-maintenance.cfg]: :)

Comment (chiyi4488) [reply etc/ncp-config.d/nc-maintenance.cfg]: Sorry, I'm not very likely to use this feature (suggested change), I've modified it. :)

**Comment by ovpc** [comment]
Nice one ! Has been added to [wiki](https://github.com/nextcloud/nextcloudpi/wiki/Configuration-Reference#nc-maintenance)

File: bin/ncp/TOOLS/nc-maintenance.sh [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+# GPL licensed (see end of file) * Use at your own risk!
+#
+# More at: https://www.cotpear.com
+# Made in Taiwan (Republic of China)
+#
+
+configure()
+{
+  [[ $ACTIVE != "yes" ]] && {
+    ncc --off
```
Comment (nachoparker) [inline bin/ncp/TOOLS/nc-maintenance.sh]:
```suggestion
    ncc maintenance:mode --off
```

File: bin/ncp/TOOLS/nc-maintenance.sh [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    return 0 
+  }
+  ncc --on
```
Comment (nachoparker) [inline bin/ncp/TOOLS/nc-maintenance.sh]:
```suggestion
  ncc maintenance:mode --on
```

Comment (chiyi4488) [reply bin/ncp/TOOLS/nc-maintenance.sh]: haha sorry

File: bin/ncp/TOOLS/nc-maintenance.sh (hunk 4 above)
Comment (chiyi4488) [reply bin/ncp/TOOLS/nc-maintenance.sh]: haha sorry

**Pull request merged**